### ISSUE TITLE
what's the actual learning_rate in data2vec2.0 ?

### ISSUE THREAD

**Issue by flishwang** [facebookresearch/fairseq#4947]
Hi,
I noticed that in the data2vec2.0 code, the losses for different samples, patches and channels are accumulated with "sum" op instead of "mean":
```
# d2v loss is first computed in func d2v_loss:
loss = F.mse_loss(x, y, reduction="none") # data2vec2.py:708
scale = 1 / math.sqrt(x.size(-1))  # data2vec2.py:715
reg_loss = loss * scale  # data2vec2.py:717

# then returned in result['losses']
result["losses"][n] = reg_loss * self.cfg.d2v_loss #

# finally reduced to scaler with "sum" in  criterions/model_criterion.py:66-75
for lk, p in losses.items():
    scaled_losses[lk] = coef * p.float().sum() # 
```

which is different from the MAE repository, which are:
```
        loss = (pred - target) ** 2
        loss = loss.mean(dim=-1)  # [N, L], mean loss per patch
        loss = (loss * mask).sum() / mask.sum()  # mean loss on removed patches
```
Therefore, for similar base learning rate (4e-4 for ViT-Large in data2vec2 vs 1.5e-4 in mae), 
the actual learning rate for data2vec2 is about (8*16*14*14*0.75*32) times compared to the "mean" reduction,
which means if the losses were averaged in data2vec2.0, the equivalent lr would be 240, which was incrediblly large.

So I suspect that my reasoning is wrong, but I can't find why.
Can you help me?
